apps/files_sharing/src/components/SharingEntryLink.vue

```diff
@@ -179,6 +193,20 @@
 
 		<!-- loading indicator to replace the menu -->
 		<div v-else class="icon-loading-small sharing-entry__loading" />
+
+		<!-- Modal to open whenever we have a QR code -->
+		<NcDialog size="normal"
+			:open.sync="showQRCode"
+			:name="title"
+			:close-on-click-outside="true"
+			@close="showQRCode = false">
+			<div class="qrCode-dialog">
```

Comment:
Do we somewhere use camel case? Seems a bit weird.
```suggestion
			<div class="qr-code-dialog">
```

Reply:
Nice catch!